File: Java/ToyORB/src/ToyORB/RemoteCallException.java
package ToyORB;

@SuppressWarnings("unused")
public class RemoteCallException extends Exception {
	public RemoteCallException(String message) {
		super(message);
	}

	public RemoteCallException(String message, Throwable cause) {
		super(message, cause);
	}

	public RemoteCallException(Throwable cause) {
		super(cause);
	}

	public RemoteCallException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
	}
}

File: .NET Core/ToyORB/RemoteCallException.cs
using System;

namespace ToyORB
{
    public class RemoteCallException : Exception
    {
        public RemoteCallException()
        {
        }

        public RemoteCallException(string message) : base(message)
        {
        }

        public RemoteCallException(string message, Exception innerException) : base(message, innerException)
        {
        }
    }
}
